tools/c7n_azure/c7n_azure/resources/vmss.py

```diff
@@ -46,6 +46,7 @@ class resource_type(ArmResourceManager.resource_type):
         default_report_fields = (
             'name',
             'location',
-            'resourceGroup'
+            'resourceGroup',
+            'sku.name'
```

Comment:
Can you get the number of nodes running?

Reply:
Yep, the field is "sku.capacity." I think a good thing to do would be able to rename default_report_field columns because they will appear as the jmespath syntax as of now